ui/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "dragon",
```

Comment:
Same comment.
Is this supposed to be hardcoded or replaced by the environment variable?

Reply:
It's angular project name and it can not be replaced by environment variables.